WalletWasabi.Fluent/Views/Wallets/MusicControlsView.axaml

```diff
@@ -5,11 +5,20 @@
              xmlns:wallets="clr-namespace:WalletWasabi.Fluent.ViewModels.Wallets"
              xmlns:i="using:Avalonia.Xaml.Interactivity"
              xmlns:behaviors="clr-namespace:WalletWasabi.Fluent.Behaviors"
+             xmlns:c="clr-namespace:WalletWasabi.Fluent.Converters"
              Background="Transparent"
              x:DataType="wallets:WalletViewModel"
              x:CompileBindings="True"
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="64"
              x:Class="WalletWasabi.Fluent.Views.Wallets.MusicControlsView">
+  <UserControl.Resources>
+    <c:ReplaceWordsByContentConverter x:Key="ReplaceWordsByContentConverter">
+      <ResourceDictionary x:CompileBindings="False">
+        <StaticResource x:Key="Play" ResourceKey="play_regular" />
+        <StaticResource x:Key="Stop" ResourceKey="stop_regular" />
```

Comment:
why also for stop?
there is no case in which it says "press stop"?

Reply:
removed for now, we can add it back once it has a message.